### DISCUSSION TITLE
Rename functionality in command line

### DISCUSSION THREAD

**Discussion by mario-amazing** [nvim-tree/nvim-tree.lua#1750] [Q&A]
How can I move rename functionality from floating preview window:
<img width="255" alt="image" src="https://user-images.githubusercontent.com/9537753/202732149-8974f076-c0f3-478c-9f43-3c2ee986821f.png">
To just command line? For better readability ( table with align and show long names)

**Comment by alex-courtis** [comment]
See `:help nvim-tree.select_prompts` to "force" ordinary vim style ex prompts.

**Reply by mario-amazing** [reply]
I set `select_prompts=true` but it didn't help. 
And I should set some kind of value to `vim.ui.select`, but don't understand what should it be

**Reply by alex-courtis** [reply]
Please accept my apologies; that option only affects selects, not input.

nvim-tree directly calls `vim.ui.input` in this case. Do you have another plugin which hijacks vim input?

**Reply by karmicdude** [reply]
> nvim-tree directly calls `vim.ui.input` in this case. Do you have another plugin which hijacks vim input?


Do you plan to add the use `select_prompts` for for operations such as renaming or creating a new node in addition to the selects? It would be great.

**Reply by mario-amazing** [reply]
> Please accept my apologies; that option only affects selects, not input.
> 
> nvim-tree directly calls `vim.ui.input` in this case. Do you have another plugin which hijacks vim input?

how can I check?

**Reply by alex-courtis** [reply]
select_prompts is currently used by:
* copy-paste
* create-file
* remove-file
* trash

**Reply by alex-courtis** [reply]
Some work is underway to enhance this prompt functionality with optional user callbacks: #1803

**Reply by alex-courtis** [reply]
> how can I check?

You could try disabling plugins other than nvim-tree. Add them back until the prompt behaviour changes.

**Reply by mario-amazing** [reply]
found -> stevearc/dressing.nvim

**Reply by alex-courtis** [reply]
> found -> stevearc/dressing.nvim

Nice one!

We should be able to add a simple recipe for this once #1803 is completed.
